src/cmd/package.go

```diff
@@ -62,7 +62,20 @@ func newPackageCommand() *cobra.Command {
 	return cmd
 }
 
-type packageCreateOptions struct{}
+type packageCreateOptions struct {
+	confirm                 bool
+	output                  string
+	differentialPackagePath string
+	setVariables            map[string]string
+	sbom                    bool
+	sbomOutput              string
+	skipSBOM                bool
+	maxPackageSizeMB        int
+	registryOverrides       map[string]string
```

Comment:
See comment above, `map[string]string` is usually an indication that there's a compound type we can name and codify.